ws/redisPubSub.js

```diff
@@ -0,0 +1,43 @@
+const config = require('../config')
+var ws = require('./sockets');
+var redis = require('redis')
+    , subscriber = redis.createClient(config.redisStore.url)
+    , r = redis.createClient(config.redisStore.url)
+
+clients = ws.clients;
+
+redisPSManager = function(socket) {
+    deviceIds = []
+    socket.on("disconnect", function() {
+        console.log('disconnected from socket!');
+        console.log('unsubscribing from deviceIds: ' + deviceIds);
+        for( var i = 0; i < deviceIds.length; i+=1) {
+            sub.unsubscribe(deviceIds[i]);
+        }
+    });
+
+    var sub = redis.createClient(config.redisStore.url);
```

Comment:
This seems like we are going to make a new redis connection for each user. Is this a good idea?